Tasks/DeployAzureResourceGroup/task.json

```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 55
+        "Patch": 56
```

Comment:
should there be a minimum agent version check to ensure the u3 agent that supports updated Register-Environment is picked?